.github/workflows/ci.yaml

```diff
@@ -73,7 +73,7 @@ jobs:
       fail-fast: false
       matrix:
         juju:
-          - agent: 2.9.45  # renovate: juju-agent-pin-minor
+          - agent: 2.9.46  # renovate: juju-agent-pin-minor
```

Comment:
I didn't get this. The latest juju 2.9/stable is 2.9.49. Why 2.9.46? CC: @carlcsaposs-canonical